app/views/items/index.html.erb

```diff
@@ -0,0 +1,55 @@
+<center>
+  <section id="item-title">
+    <% if @merchant %>
+    <h1><a href="/merchants/<%=@merchant.id%>"><%= @merchant.name %> Items</h1>
+    <% else %>
+      <h1>All Items</h1>
+    <% end %>
+  </section>
+
+<section class="boxes-container">
+  <% @items.each do |item| %>
+  <% if @merchant %>
+    <% if item.active? %>
+      <section class="single-box">
+        <div id="item-pic">
+          <%= link_to "/items/#{item.id}" do %>
+            <img src="<%=item.image %>" id="single-pic">
+          <% end %>
+        </div>
+        <div class="">
+          <p><%= link_to "#{item.name}", "/items/#{item.id}", method: :get %></p>
+          <p>description:<%= item.description %></p>
+          <p>price:<%= number_to_currency(item.price) %></p>
+          <p>inventory:<%= item.inventory %></p>
+          <p>active:<%= item.active? %></p>
+          <p><%= link_to "#{Merchant.find(item.merchant_id).name}", "/merchants/#{item.merchant_id}", method: :get %></p>
+        </div>
+      </section>
+      <% end %>
+    <% else %>
+    <section class="single-box">
+      <div id="item-pic">
+        <%= link_to "/items/#{item.id}" do %>
+          <img src="<%=item.image %>" id="single-pic">
+        <% end %>
+      </div>
+      <div class="">
+        <p><%= link_to "#{item.name}", "/items/#{item.id}", method: :get %></p>
+        <p><%= item.description %></p>
+        <p>price:<%= number_to_currency(item.price) %></p>
+        <p>inventory:<%= item.inventory %></p>
+        <p>active:<%= item.active? %></p>
+        <p><%= link_to "#{Merchant.find(item.merchant_id).name}", "/merchants/#{item.merchant_id}", method: :get %></p>
+      </div>
+    </section>
+    <% end %>
+  <% end %>
+</section>
```

Comment:
These two chunks of code within your conditional are pretty similar. Since they are so similar, instead of having different views based off of whether or not @ merchant is present, I would just put conditionals next to the lines that you don't/do want to display.